deploy/crds/jaegertracing.io_jaegers_crd.yaml

```diff
@@ -3,13 +3,24 @@ kind: CustomResourceDefinition
 metadata:
   name: jaegers.jaegertracing.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.phase
+    description: Instance's status
```

Comment:
Is this the standard description, or should it be more specific to Jaeger, e.g. `Jaeger instance's status`?

Reply:
+1. For reference: the CRD is a generated file, the source of truth is the `jaeger_types.go` file.